### PR TITLE
Sort the import and export table list based on row count

### PR DESCRIPTION
**Pull request by hbhanawat**

### CODE REVIEW

File: yb-voyager/cmd/common.go [hunk 1]
```diff
@@ -272,8 +272,14 @@ func displayExportedRowCountSnapshot(snapshotViaDebezium bool) {
 		} else {
 			addHeader(uitable, "DATABASE", "TABLE", "ROW COUNT")
 		}
-		keys := lo.Keys(exportedRowCount)
-		sort.Strings(keys)
+		keys := make([]string, 0, len(exportedRowCount))
+		for key := range exportedRowCount {
+			keys = append(keys, key)
+		}
```
Comment (sanyamsinghal) [inline yb-voyager/cmd/common.go:278]: lets use `lo.Keys()` here https://github.com/samber/lo?tab=readme-ov-file#keys

```
keys := lo.Keys(exportedRowCount)
```

File: yb-voyager/cmd/common.go [hunk 2]
```diff
@@ -394,13 +400,23 @@ func displayImportedRowCountSnapshot(state *ImportDataState, tasks []*ImportFile
 			utils.ErrExit("failed to get imported snapshot rows map: %v", err)
 		}
 	}
+	keys := make([]sqlname.NameTuple, 0, len(snapshotRowCount.Keys()))
```
Comment (sanyamsinghal) [inline yb-voyager/cmd/common.go:403]: isn't just this enough to get the keys
```
keys := snapshotRowCount.Keys()
```

File: yb-voyager/cmd/common.go (hunk 1 above)
Comment (hbhanawat) [reply yb-voyager/cmd/common.go:280]: done.

File: yb-voyager/cmd/common.go (hunk 2 above)
Comment (hbhanawat) [reply yb-voyager/cmd/common.go:405]: snapshotRowCount.Keys returns string[]. I cannot use the string object to look up in the snapshotRowCount in the slice function.

**Review by sanyamsinghal — APPROVED** [review]

**Pull request merged**